Source/WebCore/dom/ElementRareData.cpp

```diff
@@ -36,6 +36,7 @@ namespace WebCore {
 struct SameSizeAsElementRareData : NodeRareData {
     unsigned short m_childIndex;
     int m_tabIndex;
+    uint32_t contentRelevancy;
```

Comment:
But ContentRelevancyStatus  is a `uint8_t` ?

Reply:
Oops, adjusted.